app/src/forms/form/service.js

```diff
@@ -249,6 +249,7 @@ const service = {
       .modify('filterCreatedBy', params.createdBy)
       .modify('filterFormVersionId', params.formVersionId)
       .modify('filterVersion', params.version)
+      .modify('filterformSubmissionStatusCode', params.filterformSubmissionStatusCode)
```

Comment:
Is this actually going to do anything? The SubmissionMetadata views modifier will just search where the submission status code is not null.

https://github.com/bcgov/common-hosted-form-service/blob/master/app/src/forms/common/models/views/submissionMetadata.js#L22

Reply:
Yes. It will filter out draft submissions